.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "python.testing.unittestArgs": [
+        "-v",
+        "-s",
+        "./amc2moodle/tests",
+        "-p",
+        "test_*.py"
+    ],
+    "python.testing.pytestEnabled": false,
```

Comment:
Is this file really required ?

Reply:
:-D probably not witht the future new version using pyproject.toml (vscode will read it to identify tests)

Reply:
I removed it in branch new-setup-hatchling